srcs/client/src/Pages/Home/home.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from "react";
+import './home.css';
+import "../../Components/Tools/Box.css"
+import "../../Components/Tools/Text.css"
+import AuthenticatedRoute from "../../Components/services/authenticatedRoute";
+import Play from "../../Components/Tools/Button/Play";
+import { Link } from "react-router-dom";
+import "../../Components/Tools/VirtualPong/virtualPong.css"
+
+
+export default function Home () {
+
+  useEffect(() => {
+    localStorage.getItem("pathIsFree");
+    console.log("test");
+  });
+
+if (!(localStorage.getItem("pathIsFree")))
+{
+    return (
+        <>
```

Comment:
I think there's some refactoring to be done here these seem to be largely the same